medical_emr/__openerp__.py

```diff
@@ -36,57 +37,10 @@
 
     'name': 'Medical EMR : Electronic Medical Record management for Medical',
     'version': '1.0',
```

Comment:
2015-09-11 8:13 GMT-05:00 Yannick Vaucher <notifications@github.com>:

> Please change versions to 8.0.1.0


We asked to not ask for this rule until we have a clear lint, it well
generate a lot of discussione (unecesary)


-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>